react/ConditionLayoutProduct.tsx

```diff
@@ -81,6 +82,19 @@ export const HANDLERS: Handlers<ContextValues, HandlerArguments> = {
 
     return Boolean(isAvailable)
   },
+  hasMoreSellersThan({ values, args }) {
+    const { sellers } = values
+
+    const productAvailable = sellers?.filter(
+      (seller) => seller.commertialOffer.AvailableQuantity > 0
+    )
+
+    const isMoreThan =
+      sellers?.length > args?.quantity &&
+      productAvailable.length > args?.quantity
```

Comment:
`productAvailable` could be `undefined` here, right? If so the call to `productAvailable.length` would break

Reply:
This needs to be fixed

Reply:
Also, how come `sellers.length` be lesser than `productAvailable.length`? I think this is a redundant check